yt/utilities/exceptions.py

```diff
@@ -20,13 +20,12 @@ def __init__(self, filename, args=None, kwargs=None):
         self.kwargs = kwargs
 
     def __str__(self):
-        msg = f"Could not determine input format from {self.filename}"
-        if self.args is not None:
-            msg += ", {self.args}"
-        if self.kwargs is not None:
-            msg += f", {self.kwargs}"
-        msg += "."
-        return msg
+        msg = [f"Could not determine input format from `'{self.filename}'"]
+        if self.args not in (None, ()):
+            msg.append(", ".join(str(a) for a in self.args))
+        if self.kwargs not in (None, {}):
+            msg.append(", ".join(f"{k}={v}" for k, v in self.kwargs.items()))
+        return ", ".join(msg) + "`."
```

Comment:
I would prefer the message formatting to be above and then `return msg` as previously. I think it's a bit easier to read.

Reply:
adressed !